package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@youleap/types",
+  "version": "1.0.0",
+  "description": "@youleap/types for external and internal components, dashboards and functions.",
+  "license": "Apache License 2.0",
+  "repository": "@youleap/types",
+  "author": {
+    "name": "Youleap Developers",
+    "email": "developer@youleap.com",
+    "url": "https://youleap.com"
+  },
+  "types": "./index.d.ts",
+  "engines": {
+    "node": ">=14.16"
+  },
+  "scripts": {
+    "test": "xo && tsd && tsc && node script/source-files-extension.js"
+  },
+  "files": [
+    "index.d.ts",
+    "source"
+  ],
+  "keywords": [
```

Comment:
Change keywords to youleap

Reply:
Add or change?

Reply:
add or change?

Reply:
Change.
Put there what you think it's ok, but it not that important.
Its just the keywords github / npm uses to indexing and better search.